src/core/execute.c

```diff
@@ -2745,9 +2745,9 @@ void exec_context_dump(ExecContext *c, FILE* f, const char *prefix) {
                 fputs("\n", f);
         }
 
-        if (strv_length(c->inaccessible_dirs) > 0) {
+        if (strv_length(c->inaccessible_paths) > 0) {
                 fprintf(f, "%sInaccessibleDirs:", prefix);
-                strv_fprintf(f, c->inaccessible_dirs);
+                strv_fprintf(f, c->inaccessible_paths);
```

Comment:
`systemd-analyze dump` prints:
```sh
    ReadWriteDirs: ...
    ReadOnlyDirs: ...
    InaccessibleDirs: ...
```
Shouldn't we print `ReadWritePaths`, `ReadOnlyPaths`, `InacessiablePaths`?